packages/blocks/cloudhealth/src/lib/common/search-assets.ts

```diff
@@ -13,11 +13,7 @@ export async function searchAssets(
     api_version: '2',
   };
 
-  const response: any = await makeGetRequest(
-    apiKey,
-    `/api/search`,
-    queryParams,
-  );
+  const response = await makeGetRequest(apiKey, `/api/search`, queryParams);
```

Comment:
Retested the assets action, all good.

<img width="412" height="538" alt="Image" src="https://github.com/user-attachments/assets/932fc9ab-9c3f-4dae-a398-ca4415819a36" />